changelog.txt

```diff
@@ -1,5 +1,8 @@
 *** WooCommerce Subscriptions Core Changelog ***
 
+= 7.1.2 - xxxx-xx-xx =
+* Fix - Added a check for the format when listing related subscription orders.
```

Comment:
```suggestion
= 7.2.0 - xxxx-xx-xx =
* Fix - Regenerate subscriptions related order caches (renewal, resubscribe, switch) if it's stored as an invalid value to prevent fatal errors.```

I've just updated the version to 7.2.0 as we only do patch releases in `subscriptions-core` if we're going to be releasing a patch of WooCommerce Subscription :)
Also I made a small tweak to the changelog entry. Let me know your thoughts !